haskell-src/Concordium/Types/Updates.hs

```diff
@@ -978,6 +987,7 @@ extractKeysIndices p =
         BlockEnergyLimitUpdatePayload{} -> getLevel2KeysAndThreshold asParamConsensusParameters
         FinalizationCommitteeParametersUpdatePayload{} -> getLevel2KeysAndThreshold asPoolParameters
         ValidatorScoreParametersUpdatePayload{} -> getLevel2KeysAndThreshold asPoolParameters
+        CreatePLTUpdatePayload{} -> error "Not implemented yet, authorization of this update type is planned for a later iteration"
```

Comment:
Suggestion: Add a TODO comment referencing https://linear.app/concordium/issue/NOD-701/add-authorization-and-sequence-number-for-createplt-chain-updates

Reply:
Since this causes some tests to fail at present, I would suggest using a dummy implementation, such as `(Set.empty, 0)`.